src/viewer.js

```diff
@@ -93,7 +93,7 @@ export function setupViewer(container) {
   let terrainUrl;
   const ownTerrain = searchParams.get('ownterrain') !== 'false';
   if (ownTerrain) {
-    terrainUrl = 'https://terrain100.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d';
+    terrainUrl = 'https://3d.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20180601/4326/';
```

Comment:
It does not look correct to use this old timestamp.

Reply:
it is the one officially advertised on api.geo.admin.ch

Reply:
we should summon and ask to @rebert

Reply:
I will send him a mail

Reply:
Here is the URL/timestamp used by geoadmin:

GET | https://terrain101.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20200520/4326/8/269/194.terrain?v=1.0.0

It is also the one Tobias mentionned in the Cesium ticket back then.

Reply:
@gberaudo  in another NGM ticket I found a new timestamp: 20200520/4326/
I tried it and it seems to work

What do you think?

Reply:
seems the same to me